src/components/layout/header.tsx

```
@@ -24,65 +184,246 @@ export const Header = () => {
-          {isAuthenticated ? (
+    <header className="sticky top-0 z-40 border-b border-gray-200 bg-white">
+      <div className="border-b border-gray-100">
+        <div className="mx-auto max-w-[1400px] px-4 py-3 sm:px-6 sm:py-4">
+          <div className="flex items-center justify-between gap-3 sm:gap-6">
             <button
-              className="rounded-full bg-slate-900 px-4 py-2 text-sm font-semibold text-white transition hover:bg-slate-800"
-              onClick={logout}
-              type="button"
+              onClick={() => setMobileMenuOpen(!mobileMenuOpen)}
+              className="rounded-lg p-2 hover:bg-gray-100 lg:hidden"
             >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)